src/test/resources/jsonld/BT000041593.json

```diff
@@ -3,47 +3,36 @@
   "contribution" : [ {
     "type" : [ "Contribution" ],
     "agent" : {
-      "id" : "https://d-nb.info/gnd/10880724X",
       "type" : [ "Person" ],
-      "label" : "Altekamp, Heinrich",
-      "gndIdentifier" : "10880724X"
+      "label" : "Altekamp, Heinrich"
     },
     "role" : {
       "id" : "http://id.loc.gov/vocabulary/relators/aut",
       "label" : "Autor/in"
     }
   } ],
-  "coverage" : [ "Coesfeld (Kreis) | 97" ],
   "bibliographicCitation" : "Wirtschaftsspiegel. - 38 (1983) H. 7, S. 24-25 : 1 Abb.",
   "medium" : [ {
     "id" : "http://rdaregistry.info/termList/RDAproductionMethod/1010",
     "label" : "Print"
   } ],
   "spatial" : [ {
-    "id" : "https://nwbib.de/spatial#N97",
+    "id" : "https://nwbib.de/spatial#Q6210",
```

Comment:
We're getting a different ID here in `spatial.id`.

Reply:
I think this is good because we have discarded the `spatial#N...` and substituted them with `spatial#$wikidataEntityId`.